lib/chef/knife/tidy_notify.rb

```diff
@@ -65,6 +65,11 @@ def run
 
         # Fetch list of organisation names from reports directory
         org_names = reports_files.map{|r_file|r_file.split("/").last.split("_").first}.sort.uniq
+        if config[:org_list]
+          filter_orgs = config[:org_list].split(',')
+          # Take the intersection of org_names and filter_orgs
+          org_names = org_names & filter_orgs
```

Comment:
nit pick, we can use self-assignment shorthand here `org_names &= filter_orgs`

Reply:
Thanks @itmustbejj for the fix! I left comment for a minor style guide adjustment but other than that, it looks good!